live/client.py

```diff
@@ -23,6 +25,19 @@ class InvalidResponse(ElementalException):
     pass
 
 
+def etree_to_dict(tree_node):
```

Comment:
I think you can use a library for that like https://pypi.org/project/xmljson/ or https://micropyramid.com/blog/how-to-convert-xml-content-into-json-using-xmltodict/